### ISSUE TITLE
Build page for find a space prototype

### ISSUE THREAD

**Issue by cappadona** [cul-it/mann-wagon#142]
Something along the lines of this, but not reservation focused...will be fleshed out further in #143.

<img width="971" alt="screenshot 2016-04-08 10 25 09" src="https://cloud.githubusercontent.com/assets/409842/14386806/56304ada-fd74-11e5-8c10-d94300af47ea.png">

**Comment by cappadona** [comment]
At the moment, thinking I'll take this approach:

* create barebones *spaces* content type in Locomotive (`name`, `url`, `type`?)
* `url` will hold route for API request to respective system
* supplement metadata from APIs as necessary with additional fields in content type

LibCal looks promising for some minimal level of metadata that we can avoid managing in two places:
```json
{
  rooms: [
    {
      room_id: 30942,
      group_id: 10348,
      name: "Interview Room",
      capacity: 3,
      description: "Interview Room",
      directions: "Basement, Mann Library",
      image_url: ""
    }
  ]
}
```

Haven't gotten as far with Booked just yet.

**Comment by cappadona** [comment]
Booked API has even more metadata to offer...

```json
{
    "resourceId": "2",
    "name": "Room 100",
    "location": null,
    "contact": null,
    "notes": null,
    "minLength": "",
    "maxLength": "",
    "requiresApproval": 0,
    "allowMultiday": "1",
    "maxParticipants": null,
    "minNotice": "",
    "maxNotice": "",
    "description": null,
    "scheduleId": "2",
    "icsUrl": null,
    "statusId": "1",
    "statusReasonId": null,
    "customAttributes": [
        {
            "id": "3",
            "label": "Test Number",
            "value": null,
            "links": [
                {
                    "href": "**redacted**",
                    "title": "get_custom_attribute"
                }
            ],
            "message": null
        },
        {
            "id": "4",
            "label": "Test String",
            "value": null,
            "links": [
                {
                    "href": "**redacted**",
                    "title": "get_custom_attribute"
                }
            ],
            "message": null
        }
    ],
    "links": [
        {
            "href": "**redacted**",
            "title": "get_resource"
        }
    ],
    "message": null,
    "bufferTime": ""
}
```

**Comment by cappadona** [comment]
In discussion with Sara and Tobi yesterday, we decided we would **limit are reliance on the reservation systems** (LibCal & Booked) to focus on what they do best: managing reservations.

Said another way, we will opt for storing and managing the spaces metadata within Locomotive via content type(s) with a field or set of fields that identifies the reservation system and room/resource ID in the reservation system, where applicable.

**Comment by cappadona** [comment]
Not crazy about having multiple levels of nested epics (I think one level of nesting is more than enough complexity).

So backed out of the decision to make this and epic and instead will add the content type issue #156 as a sub-task for the usability task #122.

> aka **not** trying to build another Jira

**Comment by cappadona** [comment]
This was also included as part of #161. Ran into a bug using the `with_scope` tag on a relationship field, which became a bit of a time sink. The bug only affects the engine, so everything was functional when hosting the site in wagon.

Spoke with Didier on Gitter and he [confirmed it was a bug] after reviewing the details I submitted. More details on the bug and temporary workaround in 1806c47 & be55d7c.
